bigbluebutton-html5/imports/api/audio/client/bridge/sip.js

```diff
@@ -482,7 +484,7 @@ class SIPSession {
         },
         displayName: callerIdName,
         register: false,
-        userAgentString: 'BigBlueButton',
+        userAgentString: `BigBlueButton/${UA_SERVER_VERSION} (HTML5, rv:${UA_CLIENT_VERSION}) ${window.navigator.userAgent}`,
```

Comment:
Looks like we may need a git rebase -- this change was done a few weeks ago, out of scope of this PR